modular_skyrat/modules/modular_clothing/armor/armor.dm

```diff
@@ -0,0 +1,72 @@
+//TECHARMOR
+/obj/item/clothing/head/helmet/space/hardsuit/security_armor //No, this isn't spaceproof. Yes, the sprite reflects that this is not spaceproof properly. I made it a hardsuit subtype to make things easier and more readable.
+	name = "type I techhelmet"
+	desc = "A specialized exoskeleton armor helmet built into a suit of armor; offers decent protection, and comes with a flash-resistant HUD visor and headlamp."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/hats.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/head.dmi'
+	flash_protect = FLASH_PROTECTION_FLASH //The visor protects from flashes, but you're still going to go blind if you try to weld something.
+	icon_state = "hardsuit-secexo"
+	icon_state = "hardsuit0-secexo"
+	inhand_icon_state = "hardsuit0-secexo"
+	hardsuit_type = "secexo"
+	armor = list("melee" = 40, "bullet" = 30, "laser" = 30,"energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
+	clothing_flags = THICKMATERIAL | BLOCK_GAS_SMOKE_EFFECT | MASKINTERNALS
+	flags_inv = HIDEMASK|HIDEEARS|HIDEFACE|HIDEHAIR
+	mutant_variants = STYLE_MUZZLE
+
+/obj/item/clothing/suit/space/hardsuit/security_armor
+	name = "type I full-body techarmor"
+	desc = "A specialized exoskeleton armor suit, comprised of flexible protective shielding. Comes equipped with a retractable helmet which offers a flash-resistant HUD visor, along with a headlamp."
+	icon = 'modular_skyrat/master_files/icons/obj/clothing/suits.dmi'
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/suit.dmi'
+	icon_state = "hardsuit-secexo"
+	inhand_icon_state = "hardsuit-secexo"
+	max_integrity = 250
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 10, "bomb" = 25, "bio" = 50, "rad" = 0, "fire" = 55, "acid" = 70)
+	allowed = list(/obj/item/ammo_box, /obj/item/ammo_casing, /obj/item/flashlight, /obj/item/gun/ballistic, /obj/item/gun/energy, /obj/item/kitchen/knife/combat, /obj/item/melee/baton, /obj/item/melee/classic_baton/telescopic, /obj/item/reagent_containers/spray/pepper, /obj/item/restraints/handcuffs, /obj/item/tank/internals/emergency_oxygen, /obj/item/tank/internals/plasmaman) //I had to do this all snowflake style because it just would not accept any sort of global list, fucking kill me
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/security_armor
+	clothing_flags = THICKMATERIAL
+	mutant_variants = STYLE_DIGITIGRADE
+	slowdown = 0
```

Comment:
This is pretty strong for an environment suit with higher armor than the exosuit. Should this really be a default choice for security at all times. That takes away a lot of informed choice about what armor you wish to wear, and the suit still largely blocks basic level temperature dips. It'll do this anyway since space suits have thermal regulation. It just won't stop pressure damage since it no longer has that flag.

Reply:
The armor is meant to serve as a stopgap measure between being instantly killed because some chucklefuck decided to play target practice on your arms and legs and send you into hardcrit without your armor ever factoring in. Still, even so, it shares the exact same values as a normal armor vest, meaning that if you want increased protection, you now need to sacrifice the utility of your default armor for an increased amount of armor against, say, brute damage with the riot armor set. Whereas before, it was pretty much a no-brainer which armor you would choose, that being the riot suit.

As for temperature, Ill need to figure out some method of bypassing the temperature thing of hardsuits. Also, we're likely removing the cell requirement and manual thermal toggle from hardsuits entirely because its **fucking stupid.**